app/controllers/staff/users_controller.rb

```diff
@@ -53,13 +53,19 @@ def update
     @service_owner = service_owner
     @delivery_partners = delivery_partners
 
-    @user.assign_attributes(user_params)
+    reset_mfa = user_params.delete(:reset_mfa)
+    if reset_mfa
+      @user.mobile_number = nil
+      @user.mobile_number_confirmed_at = nil
+    end
+
+    @user.assign_attributes(user_params.except(:reset_mfa))
```

Comment:
An excellent find, since it highlighted that the spec for `UpdateUser` was still pending, and intended to be un-pended as part of the user edit story!

However, changing it has resulted in a few more changes to this PR, if you (or @james ) wouldn't mind having a (second) look.

Reply:
Excellent, more deletions! This all looks good.